docs/manage/_content.rst

```diff
@@ -16,7 +16,7 @@ Kolibri **Content Channel** is a collection of educational resources (video, aud
 
 .. _id_token:
 
-Each Kolibri content channel has its own **token/ID** in `Kolibri Studio <https://studio.learningequality.org/accounts/login/>`__. You can freely view and browse content to import from the **public** channels in Kolibri, but in order to import content from **private or unlisted** channels, you will need the **channel token or ID from the content curator who assembled it**.
+Each Kolibri content channel has its own **token/ID** in `Kolibri Studio <https://studio.learningequality.org/accounts/login/>`__. You can freely view and browse content to import from the **public** channels in Kolibri, but in order to :ref:`import content from private or unlisted channels <import_with_token>`, you will need the **channel token or ID from the content curator who assembled it**.
```

Comment:
> channel token or ID

maybe we should just call it the channel token? I feel like channel IDs haven't been a thing for a while now